### PR TITLE
[AKS] `az aks stop`: Add warning when private link cluster is stopped

### PR DESCRIPTION
**Pull request by levimm**
**Related command**
az aks stop: add warning when stopping a private link cluster. 

**Description**
Add warning when customer stops a private link cluster.
Similar to this change in cli extension repo: https://github.com/Azure/azure-cli-extensions/pull/7019 

**Testing Guide**

**History Notes**

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py [hunk 1]
```diff
@@ -5100,6 +5100,29 @@ def test_aks_stop_and_start(self, resource_group, resource_group_location):
...
+        # This is the only test case in which the `--generate-ssh-keys` option is enabled in the `aks create` command.
+        # Please do not enable this option in other test cases to avoid race conditions during concurrent testing.
+        # For more details, please refer to issue #18854.
+        aks_name = self.create_random_name('cliakstest', 16)
+        self.kwargs.update({
+            'resource_group': resource_group,
+            'name': aks_name
+        })
+
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} --generate-ssh-keys --enable-private-cluster'
```
Comment (FumingZhang) [inline src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py:5115]: please don't generate a new ssh key in this test case, use an existing one instead (see example in the test case below)

Comment (levimm) [reply src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py]: fixed

**Comment by levimm** [comment]
@zhoxing-ms , looks the pipeline error is not related to my change? Could you help take a look?

**Comment by yanzhudd** [comment]
> @zhoxing-ms , looks the pipeline error is not related to my change? Could you help take a look?

The CI has been fixed.

**Comment by levimm** [comment]
@yanzhudd  could you help review and merge the pr?

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**